modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/group/type/contributor/util/GroupTypeContributorUtil.java

```diff
@@ -36,19 +34,20 @@
 public class GroupTypeContributorUtil {
 
 	public static long[] getClassNameIds() {
-		Stream<GroupTypeContributor> stream = _groupTypeContributors.stream();
-
-		return ListUtil.toLongArray(
-			stream.filter(
-				GroupTypeContributor::isEnabled
-			).map(
-				GroupTypeContributor::getClassName
-			).map(
-				PortalUtil::getClassNameId
-			).collect(
-				Collectors.toList()
-			),
-			Long::valueOf);
+		List<Long> classNameIds = new ArrayList<>();
+
+		for (GroupTypeContributor groupTypeContributor :
+				_groupTypeContributors) {
+
+			if (!groupTypeContributor.isEnabled()) {
+				continue;
+			}
+
+			classNameIds.add(
+				PortalUtil.getClassNameId(groupTypeContributor.getClassName()));
+		}
+
+		return ListUtil.toLongArray(classNameIds, Long::valueOf);
```

Comment:
@jeffwu0724 Please do not use ListUtil here, use ArrayList.toArray. The ArrayList version do a array copy without iterating.